src/app/all-tickets/all-tickets.component.ts

```diff
@@ -1,15 +1,23 @@
 import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
 
 @Component({
   selector: 'app-all-tickets',
   templateUrl: './all-tickets.component.html',
   styleUrls: ['./all-tickets.component.css']
 })
 export class AllTicketsComponent implements OnInit {
-
-  constructor() { }
+  AllTickets: any;
```

Comment:
@mkorzecki Powinieneś zrobić model dla ticketów(zamiast korzystać z any), przykład **user.ts**. I to property powinno być z małej litery.